src/services.js

```diff
@@ -50,20 +50,34 @@ self.addEventListener('DOMContentLoaded', function () {
         }
     }
 
+    function GetTitle(mode) {
```

Comment:
This became duplicated as I cannot set the title in the switch above as the page is not yet loaded in that context